Firestore/Source/Core/FSTQuery.h

```diff
@@ -182,8 +182,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (nullable const model::FieldPath *)inequalityFilterField;
 
-/** Returns YES if the query has an arrayContains filter already. */
-- (BOOL)hasArrayContainsFilter;
+/**
+ * Returns the first array operator (array-contains or array-contains-any) found on a
+ * filter, or nullptr if there are no array operators.
+ */
+- (absl::optional<core::Filter::Operator>)getArrayOps;
```

Comment:
Same advice here as on C++ `Query`, but actually after merging master this file is deleted anyway.